MM2RandoLib/Resources/Asm/Auto/blocky_hitbox_fix.asm

```diff
@@ -0,0 +1,23 @@
+.include "mm2r.inc"
+
+.segment "BANKE"
+
+.org $a9c6
+	jsr CheckBlockyAlive ; Was lda $420, y
+
+.reloc
+CheckBlockyAlive:
+	; Y is Blocky index
+	lda $400, y
+	cmp #BLOCKY_OBJ_ID
+	bne +
+	
+	lda $420, y ; Obligatory
```

Comment:
Could you explain "obligatory" in the comment?

Reply:
It's the instruction that was overwritten with the jsr CheckBlockyAlive.